evm/bridge-token-factory/contracts/BridgeTokenFactory.sol

```diff
@@ -38,13 +38,14 @@ contract BridgeTokenFactory is
     address public nearBridgeDerivedAddress;
 
     mapping(uint128 => bool) public completedTransfers;
+    uint128 public currentNonce; 
```

Comment:
Maybe `initTransferNonce` or `outgoingTransferNonce`?

Reply:
Also, should we use this as a wormhole nonce as well for consistency?

Reply:
71f887f80fbdb2179c233881ef3abfeb7f2d29f3

Reply:
> Also, should we use this as a wormhole nonce as well for consistency?

IDK; I don't even understand why it is needed, it could be even 0